2996_reflection/reflection.md

```diff
@@ -3387,6 +3387,21 @@ Modify paragraph 6 to cover splicing of structured bindings:
 
 :::
 
+Modify paragraph 10.1 to prevent `*this` from being _odr-usable_ in a consteval block:
+
+::: std
+[10]{.pnum} A local entity ([basic.pre]) is _odr-usable_ in a scope ([basic.scope.scope]) if
+
+- [#.#]{.pnum} either the local entity is not `*this`, or an enclosing class[, consteval block,]{.addu} or non-lambda function parameter scope exists and, if the innermost such scope is a function parameter scope, it corresponds to a non-static member function, and
```

Comment:
Is this change necessary? `*this` isn't going to be usable anyway - we spell the lambda `[]`?

Reply:
I _think_ it is, because we're effectively saying that the lambda is only hypothetical: The consteval block scope is the controlling, actual scope.

Reply:
Actually, we probably need to say something stronger right. 
```cpp
struct C {
  void f(int x) {
    consteval {
      // it's not just that *this isn't odr-usable here
      // x isn't odr-usable here either.
    }
  }
};
```

Reply:
Yes, but that falls out by virtue of not modifying 10.2 (https://eel.is/c++draft/basic#def.odr-10.2).